Tools/SeeDot/seedot/Predictor/library_fixed.h

```diff
@@ -1042,6 +1054,62 @@ void Maxpool(TypeA* A, TypeB* B, MYINT N, MYINT H, MYINT W, MYINT C, MYINT FH, M
 	return;
 }
 
+template<class TypeA>
+void NormaliseL2(TypeA* A, MYINT N, MYINT H, MYINT W, MYINT C, MYINT scaleA, MYINT shrA) {
+	for (MYITE n = 0; n < N; n++) {
+		for (MYITE h = 0; h < H; h++) {
+			for (MYITE w = 0; w < W; w++) {
+
+				// calculate the sum square
+				int32_t sumSquare = 0;
+				MYINT shrAdiv = (1<<shrA);
+
+				for (MYITE c = 0; c < C; c++) {
+						MYINT tmp = (A[n * H * W * C + h * W * C + w * C + c] / shrAdiv);
```

Comment:
same comment as in library_fixed.cpp
use the example of matmul in this file for multiplication